modules/core/src/main/java/org/apache/ignite/internal/visor/cache/VisorCacheMetrics.java

```diff
@@ -751,6 +783,11 @@ public long getRebalancingBytesRate() {
         rebalancingBytesRate = in.readLong();
 
         qryMetrics = (VisorQueryMetrics)in.readObject();
+
+        if (protoVer > V1) {
```

Comment:
If you look at the history of this file in a git, contributors add metrics at end of serialization without change version. It makes sense to do it as well?